index.html

```diff
@@ -1854,6 +1854,53 @@ <h2>Default Language</h2>
           (e.g., en, de-AT, gsw-CH, zh-Hans, zh-Hant-HK, sl-nedis).</span>
         </span>
     </section>
+    <section id="sec-identifiers">
+      <h2>Identifiers</h2>
+      <p>
+        Web resources are identified by uniform resource identifiers (URIs).
+        The Thing Description defines the field <code>id</code> for that purpose, which is in the format "anyURI".
+        In deployments of multiple devices in the same application context, things must be distinguished by the consumer.
+        In these cases, these identifiers must be <em>unique</em>.
+        Use cases with worldwide deployments typically require <em>globally</em> unique identifiers.
+      </p> 
+      <section id="sec-identifiers-format">
+        <h3>Format</h3>
+      <p>
+        A common and well defined format for these identifiers enables easy implementation 
+        and simplifies handling across manufacturers through the use of libraries. 
+        It also ensures that there are no name clashes of devices from different manufacturers.
+        The [[UUID]] format has been developed to satisfy these requirements.
+      </p>
+        <p>
+          <span class="rfc2119-assertion" id="common-constraints-identifiers-uuid">
+            It is RECOMMENDED to use Universally Unique IDentifier (UUID) [[RFC4122]] as common format for identifers.
+          </span>
```

Comment:
I'm pleased to see that this is only a recommendation, however:
1. A UUID is not a URI. Do you perhaps mean a URN?
2. Can you explain why a URN is a better globally unique identifier than an HTTP URL with a fully qualified domain name and therefore a URN should be recommended over that?

In WebThings we still use the URL of the Thing Description as its ID, which has many benefits.

Reply:
The URL is not stable in scenarios where things may transition between different networks. Think of roaming scenarios in fleet management, IP reassignment cases etc.